quickannotator/api/v1/image/routes.py

```diff
@@ -50,29 +55,187 @@ def delete(self, args):
         return 204
 
 #################################################################################
-@bp.route('/<int:project_id>/search', endpoint="image_search")
-class ImageSearch(MethodView):
-    @bp.arguments(server_models.SearchImageArgsSchema, location='query')
-    @bp.response(200, server_models.ImageRespSchema(many=True))
-    def get(self, args, project_id):
-        """     returns a list of Images
-        """
-        result = db_session.query(db_models.Image).filter(db_models.Image.project_id == project_id).all()
-        return result, 200
-
+    @bp.route('/<int:project_id>/search', endpoint="image_search")
+    class ImageSearch(MethodView):
```

Comment:
should this be on /images/search since you're searching for images? another option is to add a filter on GET /images   , like /images?projid=id ?